contact.html

```diff
@@ -0,0 +1,75 @@
+<!DOCTYPE html>
```

Comment:
nice work, however something to note:
1. Explore design inspirations for the contact page to gain insights into what's needed.
2. Improve the layout of the input fields.

Reply:
Great work so far! A few areas to improve: consider drawing design inspiration for the contact page to better align with best practices, and refine the layout of the input fields for a more user-friendly experience.